metadrive/obs/top_down_obs_impl.py

```diff
@@ -285,12 +285,14 @@ def display(cls, lane, surface, two_side=True, use_line_color=False, color=None)
             # circular side or continuous, it is same now
             elif lane.line_types[side] == PGLineType.CONTINUOUS and isinstance(lane, CircularLane):
                 cls.continuous_curve(lane, surface, stripes_count, s0, side, color=color)
-            elif lane.line_types[side] == PGLineType.SIDE and isinstance(lane, CircularLane):
+            elif (lane.line_types[side] == PGLineType.SIDE
+                  or lane.line_types[side] == PGLineType.BARRIER) and isinstance(lane, CircularLane):
                 cls.continuous_curve(lane, surface, stripes_count, s0, side, color=color)
             # the line of continuous straight and side straight is same now
             elif (lane.line_types[side] == PGLineType.CONTINUOUS) and isinstance(lane, StraightLane):
                 cls.continuous_line(lane, surface, stripes_count, s0, side, color=color)
-            elif (lane.line_types[side] == PGLineType.SIDE) and isinstance(lane, StraightLane):
+            elif (lane.line_types[side] == PGLineType.SIDE
+                  or lane.line_types[side] == PGLineType.BARRIER) and isinstance(lane, StraightLane):
```

Comment:
`Barrier` is already used for another object. To avoid confuse, please rename it from `PGLineType.BARRIER` to  `PGLineType.GUARDRAIL`.